examples/player/simid_player.html

```diff
@@ -35,11 +35,43 @@
       simidPlayer.initializeAd();
     }
 
+    function setButtonState(disable) {
```

Comment:
can you document this function,  I'm a little confused by it.  Or better yet rename it.

Reply:
yes added comment

Reply:
How about,
setCreativeControlsState

I know it's a little more verbose but it might make more sense.

Then declare two constants at the top of this file
/** DISABLED and ENABLED constants are used to control the state of buttons. */
const DISABLED = true;
const ENABLED = false;

then when you call this function it will be
setCreativeControlsState(ENABLED);

super understandable.

Reply:
i will change the code to use consts